en-US/Players_Guide/03_Creating_a_Character/Classes/Adept/Disciple_of_the_Outcast.md

```diff
@@ -0,0 +1,50 @@
+##### Discipline of the Outcast
+
+An Outcast is an Adept that has lost their Way. Trained in the basics of the style, they have abandoned the rigorous discipline and philosophy of their tradition, and are usually now enrolled only in the school of hard knocks. Having been trained in the fundamentals of the art of focus points, these Adepts choose as often as not to apply that art to tavern brawls and street fights. Much to the horror of many more formal Adepts, their rough and tumble lifestyle tends to be a solid source of further training.
+
+> **How Outcasts Come to Be**
+> \
+> How an Adept becomes an Outcast can vary—maybe they only ever received incomplete training from a passing mentor that died or left, or maybe they were outright expelled from their Order.
+> Alternatively, at your Conductor's discretion this could be a later in life development if a player becomes disenchanted with their Martial Discipline.
```

Comment:
I moved this block up, since it was previously under the last feature of the subclass but really has to do with the flavor.